lib/json/serialization.nit

```diff
@@ -448,6 +448,25 @@ class JsonDeserializer
 	end
 end
 
+redef class Text
+
+	# Deserialize a `nullable Object` from this JSON formatted string
+	#
+	# Warning: Any deserialization errors are lost and may return `null` or a partial object.
+	# For this reason, this method should only be used in simple scripts.
+	#
+	# Require: `self` formatted in valid JSON, with deserialization metadata.
```

Comment:
a require is strong. So the current one means that if `self` is badly formated, then the program will abort. This seems to me inconsistent with the warning above it.